src/include/sof/audio/codec_adapter/codec/cadence.h

```diff
@@ -17,6 +17,16 @@
 #define LIB_NO_ERROR XA_NO_ERROR
 #define LIB_IS_FATAL_ERROR(e) ((e) & XA_FATAL_ERROR)
 
+enum cadence_stream_param {
```

Comment:
I would also have a big comment here explaining which Cadence codecs this applies to and state other may differ.